pvlib/tests/test_solarposition.py

```diff
@@ -478,21 +456,24 @@ def test_get_solarposition_pressure(pressure, expected, golden):
 
 
 @pytest.mark.parametrize("altitude, expected", [
-    (1830.14, _expected_solpos_df()),
+    (1830.14, 'expected_solpos'),
     (2000, pd.DataFrame(
         np.array([[39.88788,   50.11212,  194.34024,   39.87205,   14.64151,
                    50.12795]]),
         columns=['apparent_elevation', 'apparent_zenith', 'azimuth',
                  'elevation', 'equation_of_time', 'zenith'],
         index=['2003-10-17T12:30:30Z']))
     ])
-def test_get_solarposition_altitude(altitude, expected, golden):
+def test_get_solarposition_altitude(
+        altitude, expected, golden, expected_solpos):
     times = pd.date_range(datetime.datetime(2003, 10, 17, 13, 30, 30),
                           periods=1, freq='D', tz=golden.tz)
     ephem_data = solarposition.get_solarposition(times, golden.latitude,
                                                  golden.longitude,
                                                  altitude=altitude,
                                                  temperature=11)
+    if isinstance(expected, str) and expected == 'expected_solpos':
+        expected = expected_solpos
```

Comment:
I wish pytest could parametrize across fixtures directly instead of having to do this sort of tapdance.  FYI in cases with more than one fixture lookup like this, using `request` and `request.getfixturevalue(request.param)` can be handy: https://stackoverflow.com/a/42599627

Reply:
thanks I didn't know about that!